code/modules/gear_presets/wy.dm

```diff
@@ -47,66 +47,66 @@
 /datum/equipment_preset/wy/junior_exec
 	name = "Corporate - B - Junior Executive"
 	flags = EQUIPMENT_PRESET_EXTRA
-	paygrade = "WY-XB"
+	assignment = JOB_JUNIOR_EXECUTIVE
+	rank = JOB_JUNIOR_EXECUTIVE
+	paygrade = "WYC2"
 
 /datum/equipment_preset/wy/exec
 	name = "Corporate - C - Executive"
 	flags = EQUIPMENT_PRESET_EXTRA
-	paygrade = "WY-XC"
+	assignment = JOB_EXECUTIVE
+	rank = JOB_EXECUTIVE
+	paygrade = "WYC3"
 
 /datum/equipment_preset/wy/senior_exec
 	name = "Corporate - D - Senior Executive"
 	flags = EQUIPMENT_PRESET_EXTRA
-	paygrade = "WY-XD"
+	assignment = JOB_SENIOR_EXECUTIVE
+	rank = JOB_SENIOR_EXECUTIVE
+	paygrade = "WYC4"
 
 /datum/equipment_preset/wy/exec_spec
 	name = "Corporate - E - Executive Specialist"
 	flags = EQUIPMENT_PRESET_EXTRA
-	paygrade = "WY-XE"
+	assignment = JOB_EXECUTIVE_SPECIALIST
+	rank = JOB_EXECUTIVE_SPECIALIST
+	paygrade = "WYC5"
 
 /datum/equipment_preset/wy/exec_supervisor
 	name = "Corporate - F - Executive Supervisor"
 	flags = EQUIPMENT_PRESET_EXTRA
-	paygrade = "WY-XF"
-
-/datum/equipment_preset/wy/manager
-	name = "Corporate Manager"
-	paygrade = "WY-XG"
-	access = list()
-	skills = /datum/skills/civilian/manager
-	idtype = /obj/item/card/id/silver/clearance_badge/manager
-	headset_type = /obj/item/device/radio/headset/distress/PMC
+	assignment = JOB_EXECUTIVE_SUPERVISOR
+	rank = JOB_EXECUTIVE_SUPERVISOR
+	paygrade = "WYC6"
 
-/datum/equipment_preset/wy/manager/New()
-	. = ..()
-	access = get_all_accesses() + get_all_centcom_access()
-
-/datum/equipment_preset/wy/manager/load_gear(mob/living/carbon/human/H)
-	H.equip_to_slot_or_del(new /obj/item/clothing/under/suit_jacket/manager(H), WEAR_BODY)
-	H.equip_to_slot_or_del(new /obj/item/clothing/suit/storage/manager(H), WEAR_JACKET)
-	H.equip_to_slot_or_del(new /obj/item/clothing/head/manager(H), WEAR_HEAD)
-	H.equip_to_slot_or_del(new /obj/item/storage/belt/gun/m4a3/vp78(H), WEAR_WAIST)
-	..()
```

Comment:
you removed the parent time for all the /manager subtypes so they won't get the proper equipment, access, id type, skills or headset. it didn't have a `flags` var defines so it didn't show up in game anyway - just set the paygrade to something else